crates/rolldown_oxc_utils/src/ext/mod.rs

```diff
@@ -189,11 +167,7 @@ impl<'me, 'ast> StatementExt<'me, 'ast> for ast::Statement<'ast> {
   }
 
   fn as_module_declaration(&self) -> Option<&ast::ModuleDeclaration<'ast>> {
-    if let ast::Statement::ModuleDeclaration(module_decl) = self {
-      Some(module_decl)
-    } else {
-      None
-    }
+    self.as_module_declaration()
```

Comment:
I think we can remove `StatementExt::as_module_declaration` entirely, as `Statement` now provides this method already.